.markdownlint-cli2.yaml

```diff
@@ -13,6 +13,8 @@ config:
   max-one-sentence-per-line: true
   code-block-style: false         # not compatible with mkdocs "details" panes
   no-alt-text: false
+  MD007:
+    indent: 4
```

Comment:
mkdocs doesn't render nested lists properly unless they are double indented (4 spaces) so I've added this rule.